src/Controller/CRUDController.php

```diff
@@ -108,7 +108,7 @@ public function listAction(Request $request): Response
             return $preResponse;
         }
 
-        if ($listMode = $request->get('_list_mode')) {
+        if (null !== ($listMode = $request->get('_list_mode'))) {
```

Comment:
I'm not a fan about variable declaration in the `if`, should we split this line ?

Reply:
Yep, done